spring-cloud-task-core/src/test/java/org/springframework/cloud/task/TaskHandlerDefaultTests.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ *  * Copyright 2015 the original author or authors.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *      http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.springframework.cloud.task;
+
+import static org.junit.Assert.assertEquals;
+
+import ch.qos.logback.core.Appender;
+import org.aspectj.lang.JoinPoint;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.task.configuration.TaskHandler;
+import org.springframework.cloud.task.repository.TaskExecution;
+import org.springframework.cloud.task.util.LoggerTestUtils;
+import org.springframework.cloud.task.util.TestDefaultConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * Verifies that the TaskHandler Methods record the appropriate log header entries and
+ * result codes.
+ *
+ * @author Glenn Renfro
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = TestDefaultConfiguration.class)
+public class TaskHandlerDefaultTests {
+
+	@Autowired
+	TaskHandler taskHandler;
+
+	@Autowired
+	JoinPoint joinPoint;
+
+	@Test
+	public void testTaskException() {
+		taskHandler.beforeCommandLineRunner(joinPoint);
+		final Appender mockAppender = LoggerTestUtils.getMockAppender();
+		taskHandler.logExceptionCommandLineRunner(joinPoint);
+		LoggerTestUtils.verifyLogEntryExists(mockAppender,
+				"Updating: TaskExecution{executionId=");
```

Comment:
If we can get the `TaskExecution`, can't we verify that the execution id matches as well?